radicle-cli/src/commands/fetch.rs

```diff
@@ -0,0 +1,149 @@
+#![allow(clippy::or_fun_call)]
+use std::ffi::OsString;
+use std::path::Path;
+
+use anyhow::{anyhow, Context};
+
+use radicle::identity::doc::Id;
+use radicle::node;
+use radicle::node::{FetchResult, Handle as _, Node};
+use radicle::prelude::*;
+
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
+pub const HELP: Help = Help {
+    name: "fetch",
+    description: "Fetch repository refs from the network",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad fetch <rid> [<option>...]
+
+    By default, this command will fetch from all connected seeds.
+    To instead specify a seed, use the `--seed <nid>` option.
+
+Options
+
+    --seed <nid>    Fetch seed a specific connected peer
+    --force, -f     Fetch even if the repository isn't tracked
+    --help          Print help
+
+"#,
+};
+
+#[derive(Debug)]
+pub struct Options {
+    rid: Option<Id>,
+    seed: Option<NodeId>,
```

Comment:
How about `seeds: Vec<NodeId>` instead to specify multiple?

Reply:
Don't think it's super useful yet, was even debating having this option at all. Thing is right now you have to be connected to the seed to fetch from it.